### PR TITLE
Add CR2 support

### PR DESCRIPTION
**Pull request by gillstrom**
Added support for CR2. The first bytes are the same as TIFF, so it had to be placed before.

The extension is capitalized (`.CR2`) importing the photos from my camera, so I kept it that way. Should it be in lowercase?

### CODE REVIEW

**Comment by arthurvr** [comment]
Thanks! Needs to be added to the readme as well.

File: index.js [hunk 1]
```diff
@@ -32,6 +32,13 @@ module.exports = function (buf) {
 		};
 	}
 
+	if (((buf[0] === 0x49 && buf[1] === 0x49 && buf[2] === 0x2A && buf[3] === 0x0) || (buf[0] === 0x4D && buf[1] === 0x4D && buf[2] === 0x0 && buf[3] === 0x2A)) && buf[8] === 0x43 && buf[9] === 0x52) {
```
Comment (arthurvr) [inline index.js]: Is there a reason you added this here and not at the bottom? If there is let's add a comment about that.

File: index.js [hunk 2]
```diff
@@ -32,6 +32,13 @@ module.exports = function (buf) {
...
+		return {
+			ext: 'CR2',
+			mime: 'image/x-canon-cr2'
+		}
```
Comment (arthurvr) [inline index.js]: Missing a semicolon

**Comment by arthurvr** [comment]
> The extension is capitalized (.CR2) importing the photos from my camera, so I kept it that way. Should it be in lowercase?

It technically doesn't matter so I'd prefer to be consistent and lowercase it.

File: index.js (hunk 1 above)
Comment (gillstrom) [inline index.js]: Yeah as I said in the PR, but I'll add one in the code as well.

**Comment by arthurvr** [comment]
Can we get the filesize of the fixture down? 20MB is quite a bunch.

**Comment by kevva** [comment]
> Can we get the size of the fixture down? 20MB is quite a bunch.

CR2 files are pretty large to begin with.

**Comment by gillstrom** [comment]
Fixed everything except the file size. I have troubles finding a smaller one, since all the data is kept in the file even if I change to the size to 1x1.

**Comment by arthurvr** [comment]
I don't like it, but no way around it, so :+1:. Thanks!

**Comment by arthurvr** [comment]
@sindresorhus Can you release this?

**Comment by sindresorhus** [comment]
done

**Pull request closed without merging**